components/dashboard/src/workspaces/Workspaces.tsx

```diff
@@ -28,46 +39,121 @@ export class Workspaces extends React.Component<WorkspacesProps, WorkspacesState
-            <div className="lg:px-28 px-10 flex flex-col space-y-2">
-                <div className="py-3 flex justify-between space-x-2 text-sm text-gray-400 border-t border-b border-gray-200">
-                    <div className="w-10"></div>
-                    <div className="w-3/12">Name</div>
-                    <div className="w-4/12">Context</div>
-                    <div className="w-2/12">Pending Changes</div>
-                    <div className="w-2/12">Last Active</div>
-                    <div className="w-8"></div>
+            {wsModel && (
+                this.state?.workspaces.length > 0 || wsModel.searchTerm ?
+                    <div className="lg:px-28 px-10 flex flex-col space-y-2">
+                        <div className="px-6 py-3 flex justify-between space-x-2 text-sm text-gray-400 border-t border-b border-gray-200">
+                            <div className="w-6"></div>
+                            <div className="w-3/12">Name</div>
+                            <div className="w-4/12">Context</div>
+                            <div className="w-2/12">Pending Changes</div>
+                            <div className="w-2/12">Last Active</div>
+                            <div className="w-8"></div>
+                        </div>
+                        {
+                            wsModel.active || wsModel.searchTerm ? null :
+                                <div className="whitespace-nowrap flex space-x-2 py-6 px-6 w-full justify-between bg-gitpod-kumquat-light rounded-xl">
+                                    <div className="pr-3 self-center w-6">
+                                        <div className="relative rounded-full w-3 h-3 bg-red">
+                                            <div className="absolute text-xs text-gray-100"
+                                                style={{
+                                                    left: "5px",
+                                                    top: "-2px"
+                                                }}
+                                                ref={(node) => {
+                                                    if (node) {
+                                                        node.style.setProperty("font-size", "10px", "important");
+                                                    }
+                                                }}>!</div>
+                                        </div>
+                                    </div>
+                                    <div className="flex-1 flex flex-col overflow-x-auto">
+                                        <div className="text-red font-semibold">Garbage Collection</div>
+                                        <p className="text-gray-500">Unpinned workspaces that have been stopped for more than 14 days will be automatically deleted. <a className="text-blue-500 underline" href="https://www.gitpod.io/docs/life-of-workspace/#garbage-collection">Learn more</a></p>
```

Comment:
**issue**: The garbage collection message should remain when searching _All_ workspaces. In the future we could deselect a filter or have a more clear searxh state when searching and remove the GC message. For example, the GV message could include the state of the search (e.g Search Results for `xyz`). 🔍

Reply:
I believe when I search for a specific workspace, I have already seen the warning and really would like to be able to focus on the search result. WDYT?

Reply:
Agree but I'd say it depends on if we hold or release the filter state on the status.

**thought**: For example, if the filter status persists and search happens only for the _All_ workspaces it wouldn't hurt to persist the GC message. If on search, we release the filter state and search across all workspaces it would make more sense to remove the GC message. Since the second filter option is _All_ here it's easier to remove the GC message here. Let's do it!